src/k8s-engine/k3sHelper.ts

```diff
@@ -503,6 +506,24 @@ export default class K3sHelper extends events.EventEmitter implements VersionLis
         writeStream.end(userYAML, 'utf-8');
       });
       await fs.promises.rename(workPath, userPath);
+
+      // The config file we modified might not be the top level one.
+      // Update the current context.
+      console.log('Setting default context...');
+      await new Promise<void>((resolve, reject) => {
+        const child = childProcess.spawn(
+          resources.executable('bin/kubectl'),
+          ['config', 'use-context', contextName],
+          { stdio: 'inherit' });
+
+        child.on('error', reject);
```

Comment:
Are there no args that accompany the `error` signal that can be passed to the `reject`?

Reply:
There is! It's passed _directly_ to `reject`, because it's the first argument.  (The callback is `reject`, not `() => rejcet()`.)